extensions/amp-list/0.1/amp-list.js

```diff
@@ -387,15 +390,41 @@ export class AmpList extends AMP.BaseElement {
           AmpEvents.DOM_UPDATE, /* detail */ null, {bubbles: true});
       this.container_.dispatchEvent(event);
 
-      // Change height if needed.
-      this.measureElement(() => {
-        const scrollHeight = this.container_./*OK*/scrollHeight;
-        const height = this.element./*OK*/offsetHeight;
-        if (scrollHeight > height) {
-          this.attemptChangeHeight(scrollHeight).catch(() => {});
-        }
-      });
+      if (!isExperimentOn(this.win, 'amp-list-resize')) {
+        // Change height if needed.
+        this.measureElement(() => {
+          const scrollHeight = this.container_./*OK*/scrollHeight;
+          const height = this.element./*OK*/offsetHeight;
+          if (scrollHeight > height) {
+            this.attemptChangeHeight(scrollHeight).catch(() => {});
+          }
+        });
+      }
     }, this.container_);
+
+    if (isExperimentOn(this.getWin(), 'amp-list-resize')) {
```

Comment:
possible to move this inside the existing `mutateElement` block above and remove the second `mutateElement`?

Reply:
This is because it mutates a different thing. By removing the `sizer`, we are mutating `this.element`, whereas the other mutate block is contained to `this.container_`.

Reply:
got it.

Reply:
You can remove `this.container_` param from the mutate call and then combine the blocks. A mutate pass encompasses children as well.

Reply:
I originally intended to preserve the limited scope because if it's size-defined, it won't need to mutate the entire element. But since the proposed experiment is to change it to be layout container... ¯\_(ツ)_/¯. 

Fixed to combine both mutate blocks.